cms/static/js/collections/asset.js

```diff
@@ -37,8 +37,7 @@ define([
             return PagingCollection.prototype.parse.call(this, response, options);
         },
 
-        /* jshint unused:false */
-        parseState: function (response, queryParams, state, options) {
+        parseState: function(response, queryParams, state, options) {  // eslint-disable-line no-unused-vars
```

Comment:
Is the space between function and ( removed part of this autofix? The bootstrap-modal and transition js files below didn't get this lint.

LGTM, 👍